### ISSUE TITLE
Node.js 12 actions are deprecated

### ISSUE THREAD

**Issue by vol1ura** [miloserdow/capistrano-deploy#45]
Hello, I have deprecation notice 

```
Node.js 12 actions are deprecated. For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/. Please update the following actions to use Node.js 16: miloserdow/capistrano-***
```

**Comment by mateuszgorniak** [comment]
I see it has been fixed by https://github.com/miloserdow/capistrano-deploy/pull/44 but it's not released as a separate tag. Could you do it @miloserdow ? :-)

**Comment by miloserdow** [comment]
> I see it has been fixed by #44 but it's not released as a separate tag. Could you do it @miloserdow ? :-)

Oh, just noticed it. Will do the release this week.

**Comment by miloserdow** [comment]
Done
